public/js/selfoss-base.js

```diff
@@ -20,13 +20,18 @@ var selfoss = {
         source: '',
         ajax: true
     },
-
+    
```

Comment:
I understand the coding isn’t really that important but let’s not make it worse please.